src/LOAuth.ts

```diff
@@ -300,9 +300,10 @@ class LOAuth {
       } else {
         console.warn(error, 'Error refreshing access token - redirecting');
         if (this.clientOptions.loginRedirectUri) {
+          const hasQueryParams = this.clientOptions.loginRedirectUri?.split('?')[1];
```

Comment:
Looks ok to me - this package also pulls in a `query-string` library - I'm not sure if it has some util to check for query parameters.  This seems fine though